junit5/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.togglz</groupId>
+    <artifactId>togglz-project</artifactId>
+    <version>2.6.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>togglz-junit5</artifactId>
+  <name>Togglz - JUnit 5 integration</name>
+  <description>Togglz - JUnit 5 integration</description>
+
+  <properties>
+    <!-- Minimal Java version -->
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+
+    <junit.jupiter.version>5.0.0</junit.jupiter.version>
+    <junit.platform.version>1.0.0</junit.platform.version>
+  </properties>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-testing</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-junit</artifactId>
```

Comment:
I wonder if this is problematic. Actually having both `togglz-junit` and `togglz-junit5` on the classpath seems weird. Especially because you will get all sorts of `togglz-junit` classes on your classpath although using them would fail because junit4 is missing. This may be confusing for users.

Thinking about this more: Wouldn't it make sense to move the `VariationSet` and `VariationSetBuilder` classes to `togglz-testing`? This way `togglz-junit5` could simply depend on this common testing module just like `togglz-junit`. AFAIK those classes actually doesn't depend on any JUnit API and therefore actually don't belong into the `togglz-junit` module. What do you think?

Reply:
I agree, these classes should go to `togglz-testing`, they don't belong to JUnit. But into which package, `org.togglz.testing` or `org.togglz.testing.vary`?
Anyway, all projects using these classes need to adjust the imports in their (test) code. I think this should be acceptable.

Reply:
I would prefer the package `org.togglz.testing.vary`.

But yes, your are correct. Moving these classes would break existing user tests which depend on these classes. I don't think that we can do that in a minor release. But perhaps we could use some tricks to preserve compatibility:

The `VariationSet` from `togglz-junit` could just extend the `VariationSet` from `togglz-testing`. Something like that:

```java
public interface VariationSet<F extends Feature> extends org.togglz.testing.vary.VariationSet<F>{}
```

And `VariationSetBuilder` from `togglz-junit` could just be an empty class with just a single static `create()` method which returns an instance of `VariationSetBuilder` from `togglz-testing`. The `create` method is the only way to create instances of this class, so this may work.

Then we could simply deprecate both classes from `togglz-junit` and therefore won't break user tests. Do you think this could work? That's just my thought without having a deeper look at the classes.

Reply:
Sounds like a good idea, I will implement it this way.